integration-tests/src/test/kotlin/org/stellar/anchor/platform/test/Sep24End2EndTests.kt

```diff
@@ -11,7 +11,6 @@ import kotlin.time.Duration.Companion.seconds
 import kotlinx.coroutines.delay
```

Comment:
💡 Can we separate the fix to a different PR?

Reply:
I needed to add tests for SEP-6 client status callbacks which ended up requiring touching the SEP-24 tests.

Reply:
These changes will be thrown away once I merge in `develop`, anyway. I've only modified these tests so the CI passes.